packages/flutter/lib/src/material/flexible_space_bar.dart

```diff
@@ -205,6 +216,32 @@ class _FlexibleSpaceBarState extends State<FlexibleSpaceBar> {
     };
   }
 
+  EdgeInsetsGeometry _getTitlePadding(
+    double t,
```

Comment:
"t" is not descriptive

Reply:
Fair enough this is true. Also here the is another method in the class "_getCollapsePadding" which also just passed the time attribute as 't'.  Do you have any suggestions how to call this?

Reply:
Ok i change the progress 't' now to 'expansionFactor'. Let me know if you are ok with this?